isb_web/main.py

```diff
@@ -107,7 +107,9 @@
 @app.on_event("startup")
 def on_startup():
     dao.connect_sqlmodel(isb_web.config.Settings().database_url)
-    orcid_ids = sqlmodel_database.all_orcid_ids(dao.get_session())
+    session = dao.get_session()
+    orcid_ids = sqlmodel_database.all_orcid_ids(session)
+    session.close()
```

Comment:
This was an existing bug on app startup.  We don't have the dependency injection at that point so just manually open and close the session here.